server/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -789,7 +791,7 @@ public void testInnerJoinSyntaxDoesNotExtendsWhereClause() throws Exception {
             "select * from users inner join users_multi_pk on users.id = users_multi_pk.id");
         assertThat(mss.where(), isLiteral(true));
         assertThat(mss.joinPairs().get(0).condition(),
-            isSQL("(doc.users.id = doc.users_multi_pk.id)"));
+                   isSQL("(doc.users.id = doc.users_multi_pk.id)"));
```

Comment:
Why all the formatting changes in this file?

Could you revert them to keep the scope of the PR narrower?